pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -49,9 +49,9 @@
     .thegap{style: "height: 1px"}
       &nbsp;
     #desktop-header.desktop-feature
-      = view :header, variant: "desktop"
-    #mobile-header.mobile-feature
-      = view :header, variant: "mobile"
```

Comment:
It looks like there are a couple of places that do this: https://github.com/code-dot-org/code-dot-org/search?l=Haml&q=variant&utf8=%E2%9C%93

Should we update up all of these instances too? And delete all the variant stuff? Maybe @breville has insight as to why we needed to duplicate this on /learn?